### ISSUE TITLE
Colors lag after change of Variable or Base map in GreatLakes_Dashboard

### ISSUE THREAD

**Issue by hdsingh** [reproducible-notebooks/COAWST-ROMS_Dashboards#7]
The colors seem to lag by one after changing the `Model Variables` or `Basemap` i.e the color of location marker does not match its corresponding line color. I am trying to figure out the reason behind this. I have also tried to replace the `get_line_color` with `color_stream` but the behaviour still persists. Can you please look into this and help me understand why this might be happening?

**Comment by rsignell-usgs** [comment]
@hdsingh, who were you hoping would look into this?

**Comment by hdsingh** [comment]
I was hoping if any one  you, @martindurant  or @jsignell could help me figure this out. Thanks for letting me know. From next time I would be careful in directly mentioning those who can help.

**Comment by martindurant** [comment]
Sorry, could you perhaps describe what you are seeing in a bit more detail? Which notebook and and what point? Perhaps a screenshot might be appropriate. If it's a genuine issue, it might be appropriate to report at Panel or Holoviews.

**Comment by hdsingh** [comment]
@rsignell-usgs @martindurant Here is more detail of the issue. Please let me know if anything else is required.

The issue is in GreatLakes_Dashboard. The dashboard formed works well initially. However when we change anyone of `Model Variables` or `Basemap`, lag of color of a point from its respective line starts. 

Initially , when color sync is perfect (`Model variable` is `zeta`):
![1](https://user-images.githubusercontent.com/30382331/57031337-203ee400-6c65-11e9-859e-4293432388d2.jpeg)

but later on (`Model variable` is `temp`):

![2](https://user-images.githubusercontent.com/30382331/57031350-292fb580-6c65-11e9-99eb-504df32084a7.jpeg)
The color points are displayed  in order of `orange`, `yellow`,`blue` while the time series lines are displayed in order of `red`, `orange`, `yellow`. 
`Red` is the color  provided in the `exception` in `get_line_color` function which is being triggered.

One thing to note here is that before variable change, location points are displayed just a little ahead of displaying time series curve while later on, the time series curve is displayed early as compared to the location points.

I am unable to find the reason because of which it does not runs well on changing the varibles.


Also the generated time series map does not automatically adjusts itself. We need to zoom out manually to see all the lines.

**Comment by rsignell-usgs** [comment]
> Also the generated time series map does not automatically adjusts itself. We need to zoom out manually to see all the lines.

@hdsingh, let's try to not mix issues.  I encountered this as well.

Can you please raise this as a new issue?

**Comment by hdsingh** [comment]
Yes sir, [here is the separate issue](https://github.com/reproducible-notebooks/COAWST-ROMS_Dashboards/issues/10) regarding this.
